app/views/help/new_features.md

```diff
@@ -0,0 +1,8 @@
+<!-- When updating this file, update date to flag users of new content -->
```

Comment:
It's the help article. When that's updated, the notice on the main page
shows up.

On Monday, August 15, 2016, Robb Shecter <notifications@github.com> wrote:

> In app/views/help/new_features.md
> <https://github.com/18F/C2/pull/1463#discussion_r74829399>:
>
> > @@ -0,0 +1,8 @@
> > +<!-- When updating this file, update date to flag users of new content -->
>
> What's this file for? Like, a scratch pad for code we might use in the
> future?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/18F/C2/pull/1463/files/e4934213ad73b6ef57602199435025efc572dc31#r74829399>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AFDQeAcHLZHGzbt6HYTHTWTcfQNc_CMNks5qgMy9gaJpZM4JjiQb>
> .
>


-- 
Nick Bristow
18F Innovation Specialist
Nick.Bristow@gsa.gov